src/Webapi/Dom/Webapi__Dom__DataTransferItemList.res

```diff
@@ -0,0 +1,8 @@
+type t
+
+@get external length: t => int = "length"
+@get_index external get: (t, int) => option<Webapi__Dom__DataTransferItem.t> = ""
+@send @return(nullable) external addString: (t, string, string) => option<Webapi__Dom__DataTransferItem.t> = "add"
```

Comment:
seems like a good place to put named arguments - I would never guess the first `string` is the data and the second is the _content type_

Reply:
Yeah, added that and to setData as well since it can also be incorrect in ordering.